components/Tabs/examples/TabBarIconExample.m

```diff
@@ -52,39 +54,41 @@ - (void)incrementBadges:(id)sender {
 #pragma mark - Private
 
 - (void)loadTabBar {
-  const CGRect bounds = self.view.bounds;
+  self.tabBar = [MDCTabBar new];
+  self.tabBar.delegate = self;
+  self.tabBar.alignment = MDCTabBarAlignmentCenterSelected;
 
-  // Short tab bar with a small number of items.
   NSBundle *bundle = [NSBundle bundleForClass:[self class]];
   UIImage *infoImage =
-      [UIImage imageNamed:@"TabBarDemo_ic_info" inBundle:bundle compatibleWithTraitCollection:nil];
+  [UIImage imageNamed:@"TabBarDemo_ic_info" inBundle:bundle compatibleWithTraitCollection:nil];
   UIImage *starImage =
-      [UIImage imageNamed:@"TabBarDemo_ic_star" inBundle:bundle compatibleWithTraitCollection:nil];
-  self.tabBar =
-      [[MDCTabBar alloc] initWithFrame:CGRectMake(0, 0, CGRectGetWidth(bounds) - 20.0f, 0)];
-  self.tabBar.center = CGPointMake(CGRectGetMidX(self.view.bounds), 150);
+  [UIImage imageNamed:@"TabBarDemo_ic_star" inBundle:bundle compatibleWithTraitCollection:nil];
   self.tabBar.items = @[
-    [[UITabBarItem alloc] initWithTitle:@"Two" image:infoImage tag:0],
-    [[UITabBarItem alloc] initWithTitle:@"Tabs" image:starImage tag:1]
-  ];
+                        [[UITabBarItem alloc] initWithTitle:@"Info" image:infoImage tag:0],
+                        [[UITabBarItem alloc] initWithTitle:@"Stars" image:starImage tag:0]
```

Comment:
the indentation got weird here - could the first item be on the first line, like '@[[UITabBarItem…'?